tangelo/tangelo/__main__.py

```diff
@@ -194,6 +189,19 @@ def main():
         print tangelo_version
         return 0
 
+    # Make all logging go to stdout instead of stderr.
+    #
+    # The "cherrypy.error" logger contains two handlers by default - a
+    # NullHandler, and a StreamHandler that writes to stderr.  This piece of
+    # code simply removes the StreamHandler, modifies it to write to stdout, and
+    # then replaces it.
+    import logging
+    logger = logging.getLogger("cherrypy.error")
+    handler = logger.handlers[1]
+    logger.removeHandler(handler)
+    handler.stream = sys.stdout
+    logger.addHandler(handler)
+
```

Comment:
Just to be clear, this part of the code doesn't merge the streams, it just causes *all* logging to be sent to ``stdout``.  Anything written purposefully to ``stderr`` will still wind up on that stream.  Merging the streams with ``sys.stderr = sys.stdout`` is a "nuclear option" that I also don't like.

That being said, the tests can just be written to examine the "correct" stream (if, upon writing a test, it fails for this reason, we simply change the test to examine the other one) but this seems like a fragile practice to me.

The other option is to change the code a bit so all logging goes to ``stderr`` instead.  The fact that Girder happens to use a ``print`` statement need not overly concern us, since we can't control where other software writes its output anyway.

I just don't like having the log output go to different streams at different times.  It is confusing and it will cause someone (a future user or developer) a headache at some point.